### PR TITLE
Content modelling/744 confirmation

### PR DESCRIPTION
**Pull request by Harriethw**
~~DO NOT MERGE - unless UR is not happening?~~ Merging after 5:30pm

https://trello.com/c/Japx95Dw/744-update-confirmation-screen-for-create-journey
https://trello.com/c/F5ei1nZ9/749-update-confirmation-screens-for-edit-journey

### mural 

Screenshot 2024-12-04 at 16 46 49

 
### code
Screenshot 2024-12-04 at 16 45 59

### mural 
Screenshot 2024-12-04 at 16 47 34

 
### code
Screenshot 2024-12-04 at 16 45 31

### mural 

Screenshot 2024-12-04 at 16 47 38
 
### code

Screenshot 2024-12-04 at 16 44 08

----

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

**Review by pezholio — APPROVED** [review]
This LGTM - I would be happier if that logic in the first commit was in a helper, but in the interests of getting this out before tomorrow, let's go with what we've got for now 👍

File: lib/engines/content_block_manager/app/controllers/content_block_manager/content_block/editions/workflow_controller.rb
Comment (Harriethw) [reply lib/engines/content_block_manager/app/controllers/content_block_manager/content_block/editions/workflow_controller.rb:81]: that's a good shout, was thinking that it was a lot of annoying logic - I'll do it in another PR just to avoid holding this one up

Comment (Harriethw) [reply lib/engines/content_block_manager/app/controllers/content_block_manager/content_block/editions/workflow_controller.rb:81]: ok, I've tried for a bit to put this into a helper method in a clean way - but because it needs the edition and the is_scheduled param it makes it a bit more complicated than I thought (would the view have to pass those variables to the helper method, because controllers aren't meant to call helpers according to the docs?) in javascript land I would just stick in a utils, not sure what to do here that wouldn't make it more complicated...

Comment (pezholio) [reply lib/engines/content_block_manager/app/controllers/content_block_manager/content_block/editions/workflow_controller.rb:81]: One approach could be using the presenter pattern, have a class in `app/presenters` that you initialize in the controller, then pass to the view

**Pull request merged**
